types/account/index.d.ts

```diff
@@ -81,6 +79,13 @@ export interface AccountSettingsPrefs {
 	zimbraPrefIncludeTrashInSearch?: BooleanString;
 	zimbraPrefOutOfOfficeFromDate?: GeneralizedTime;
 	zimbraPrefOutOfOfficeUntilDate?: GeneralizedTime;
+	zimbraPrefHtmlEditorDefaultFontColor?: string;
+	zimbraPrefHtmlEditorDefaultFontFamily?: string;
+	zimbraPrefHtmlEditorDefaultFontSize?: string;
+	zimbraPrefLocale?: string;
+	zimbraPrefMailPollingInterval?: string;
```

Comment:
type can be improved. zimbraPrefMailPollingInterval is declared of type duration, which can be translated in
```ts
export type DurationUnit = 'd' | 'h' | 'm' | 's' | 'ms';
export type Duration = `${number}${DurationUnit | ''}`;
```

Then, then cast can be removed from here:
https://github.com/Zextras/carbonio-shell-ui/blob/a9824c0f2441079bf95cebb43a6524437f89b30a/src/store/network/utils.ts#L28